documentation/install-guide.html

```diff
@@ -191,158 +296,243 @@ <h3><a name="autosetup"></a>Autosetup</h3>
+                <figcaption>Windows Service commands in a console windows.</figcaption>
+            </figure>
+            <p>
+                As of 4.1.5 Openfire installs the service automatically. But if you are using older version or want to
+                reinstall the service, you can use the <code>openfire-service.exe</code> executable in the
+                <code>bin</code> directory of the installation to install or uninstall the service.
+            </p>
+            <p>
+                From a console window, you can run the following commands:
+            </p>
+            <dl>
+                <dt><code>openfire-service /install</code></dt>
+                <dd>installs the service.</dd>
+
+                <dt><code>openfire-service /uninstall</code></dt>
+                <dd>uninstalls the service.</dd>
+
+                <dt><code>openfire-service /start</code></dt>
+                <dd>starts the service</dd>
+
+                <dt><code>openfire-service /stop</code></dt>
+                <dd>stops the service.</dd>
+            </dl>
+            <p>
+                You can also use the Services tool in the Windows Control Panel to start and stop the service.
+            </p>
+            <p>
+                <b>Note:</b> the graphical launcher is not compatible with the Windows service. If you install the
+                service, you should use service controls as described above to control the server rather than the
+                graphical launcher.
+            </p>
+            <p>
+                <b>Upgrading Note:</b> you should stop Openfire service before running the installer to upgrade to the
+                newer version. Starting with 4.1.5 version Openfire will automatically stop and reinstall the service
+                when upgrading with the installer.
+            </p>
+
+        </section>
+
+        <section>
+
+            <h3>Custom Parameters</h3>
+
+            <p>
+                Advanced users may wish to pass in parameters to the Java virtual machine (VM) to customize the runtime
+                environment of Openfire. You can do this by creating a <code>openfire.vmoptions</code> file in the
+                <code>bin/</code> directory of your Openfire installation. For the Windows service, you'd create a new
+                text file called <code>openfire-service.vmoptions</code>.
+            </p>
+
+            <p>
+                Each parameter to the VM should be on a new line of the file. For example, to set the minimum heap size
+                to 512 MB and max VM heap size to 1024 MB, you'd use:
+            </p>
+
+            <pre><code>-Xms512m
+-Xmx1024m</code></pre>
+
+            <p>
+                To create parameters for the normal launcher, create a file called <code>openfired.vmoptions</code>
+                (since the openfire.exe launcher invokes the openfired.exe executable to actually start the server).
+            </p>
```

Comment:
Probably out of scope, but it's not clear which file I would normally want to add to increase the memory cap for a running instance, and I'd probably punt and set all three.

Reply:
Doesn't hurt to do that :) I'm not sure how to improve this without adding so much text that it becomes unreadable. Feel free to add a change, but I'm happy to keep this as-is.